assets/templates/aws.sqs/sqs.gotext.log

```diff
@@ -0,0 +1,2 @@
+{ "create" : { "_index": "metrics-aws.sqs-default" } }
```

Comment:
Does everyone that builds a template have to add this? I would assume we can automatically add this based on the knowledge from the dataset or if not, have a config that links to the different templates for generation and has it in.

This might also solve the issue below. If there is just 1 event per doc and we know it is json (could be indicated) we can munch it into a single line afterwards.

Reply:
> Does everyone that builds a template have to add this?

Definitely no: I just added it here in the draft (while it might stick for a few iterations)
Two considerations:
1. I still have to investigate if that line is required when loading a bulk request payload from a locally file in rally
2. Once we will have support for different outputs, knowing that we will generate rally output will add the line automatically